lmu2png/src/main.cpp

```diff
@@ -118,22 +130,25 @@ void DrawTiles(SDL_Surface* output_img, stChipset * gen, uint8_t * csflag, std::
 	}
 }
 
-void DrawEvents(SDL_Surface* output_img, stChipset * gen, std::unique_ptr<lcf::rpg::Map> & map, int layer) {
+void DrawEvents(SDL_Surface* output_img, stChipset * gen, std::unique_ptr<lcf::rpg::Map> & map, int layer, bool ignore_conditions, bool simulate_movement) {
 	for (const lcf::rpg::Event& ev : map->events) {
 		const lcf::rpg::EventPage* evp = nullptr;
 		// Find highest page without conditions
-		for (int i = 0; i < (int)ev.pages.size(); ++i) {
-			const auto& flg = ev.pages[i].condition.flags;
-			if (flg.switch_a || flg.switch_b || flg.variable || flg.item || flg.actor || flg.timer || flg.timer2)
-				continue;
-			evp = &ev.pages[i];
+		if (ignore_conditions)
+			evp = &ev.pages[0];
+		else {
+			for (int i = 0; i < (int)ev.pages.size(); ++i) {
+				const auto& flg = ev.pages[i].condition.flags;
+				if (flg.switch_a || flg.switch_b || flg.variable || flg.item || flg.actor || flg.timer || flg.timer2)
+					continue;
+				evp = &ev.pages[i];
+			}
 		}
 		if (!evp)
-			continue;
+				continue;
```

Comment:
Please check for unnecessary whitespace changes.

Reply:
my bad